src/Result.ts

```diff
@@ -19,7 +24,8 @@ export default class Result {
    * Run Axe on a browser page that is displaying a story.
    */
   static async fromPage(page: Page, story: ProcessedStory) {
-    const disabledRules = [...defaultDisabledRules, ...story.disabledRules];
+    const disabledRulesFromStory = story.disabledRules || [];
```

Comment:
Does separating out `disabledRulesFromStory` make this more readable? It's cool if it does, just to check what we're doin here 😸 

Also nit: we don't need to provide a fallback `[]`, since it's perfectly legit to spread `undefined`.

```suggestion
    const disabledRulesFromStory = story.disabledRules;
```

Reply:
huh, I added this because I was getting `Type 'string[] | undefined' must have a '[Symbol.iterator]()' method that returns an iterator.ts(2488)` with the updated types

![image](https://user-images.githubusercontent.com/14186239/181648867-73d49249-299e-4f54-b475-353158af6116.png)

Reply:
Weird. That error doesn't make sense to me, since it's legit JS. Made me realize something, though - 

Before this PR the parameters were all **_required_** in ProcessedStory: https://github.com/chanzuckerberg/axe-storybook-testing/blob/d468667b74cdab25c97d4743699de11f8964572c/src/ProcessedStory.ts#L4-L10

but this PR changes them to all be **_optional_**.

I don't think we want to change the types of the params in ProcessedStory to be optional. The whole point of ProcessedStory is that everything is "normalized" and non-optional.